src/config.js

```diff
@@ -29,8 +29,7 @@ const path = require('path');
  * @param {String} [fileName = .tao-extension-release] - the config file
  * @returns {config} the config manager
  */
-module.exports = function configFactory(dir = process.env.HOME, fileName = '.tao-extension-release'){
-
+module.exports = function configFactory(dir = process.env[(process.platform == 'win32') ? 'USERPROFILE' : 'HOME'], fileName = '.tao-extension-release'){
```

Comment:
great, but after checking the recommended way is seems to be using [os.homedir()](https://nodejs.org/dist/latest-v8.x/docs/api/os.html#os_os_homedir) : 

```
const os = require('os');
module.exports = function configFactory(dir = os.homedir(), fileName = '.tao-extension-release'){
 ///...
```
 Could you please give this a try when you have a few minutes